frontend/packages/otr/public/i18n/en-GB/common.json

```diff
@@ -7,6 +7,7 @@
       "appNameAbbreviation": "OTR",
       "appTitle": "Register of legal interpreters | Finnish National Agency for Education",
       "back": "Back",
+      "backToHomePage": "Back to first page",
```

Comment:
Pitäiskö tätä pyytää re-checkkiin?

Reply:
Joo. Tai korjattaisko tuossa muodossa tai muodossa `Back to home page` suoraan sekä tänne että AKR:ään?

Reply:
Ehkä "Back to front page" vois olla hyvä, vastaa AKR:n ja OTR:n mallia.